### PR TITLE
Remove the unnessary empty line after label

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary
Code refactor only:

- Remove the unnessary empty line after label
- Return -EINVAL for the internal API

## Impact
Minor

## Testing
Pass CI, please ignore mix case warning.

### CODE REVIEW

File: arch/arm/src/cxd56xx/cxd56_gnss.c [hunk 1]
```diff
@@ -2787,8 +2784,8 @@ static ssize_t cxd56_gnss_read(struct file *filep, char *buffer,
 
   ret = fw_gd_readbuffer(type, offset, buffer, len);
 
-  _err:
-  _success:
+err:
+success:
```
Comment (pkarashchenko) [inline arch/arm/src/cxd56xx/cxd56_gnss.c:2788]: Maybe we can use a single `out` label here?

File: arch/arm/src/cxd56xx/cxd56_gnss.c [hunk 2]
```diff
@@ -3063,21 +3060,20 @@ static int cxd56_gnss_register(const char *devpath)
...
       cxd56_cpu1sigregisterhandler(devsig_table[i].sigtype,
                                    devsig_table[i].handler);
     }
 
   gnssinfo("GNSS driver loaded successfully!\n");
-
   return ret;
 
-  _err2:
+err2:
```
Comment (pkarashchenko) [inline arch/arm/src/cxd56xx/cxd56_gnss.c:3073]:
```suggestion
err1:
```

File: arch/arm/src/stm32f7/stm32_bbsram.c [hunk 3]
```diff
@@ -424,7 +424,7 @@ static off_t stm32_bbsram_seek(struct file *filep, off_t offset,
    *   point, subsequent reads of data in the gap shall return bytes with the
    *   value 0 until data is actually written into the gap."
    *
-   * We can conform to the first part, but not the second.  But return EINVAL
+   * We can conform to the first part, but not the second. But return -EINVAL
```
Comment (pkarashchenko) [inline arch/arm/src/stm32f7/stm32_bbsram.c:427]:
```suggestion
   * We can conform to the first part, but not the second.  But return -EINVAL
```
I think most of the comments have 2 spaces after dot

File: arch/arm/src/stm32h7/stm32_bbsram.c [hunk 4]
```diff
@@ -470,7 +470,7 @@ static off_t stm32_bbsram_seek(struct file *filep, off_t offset,
    *   point, subsequent reads of data in the gap shall return bytes with the
    *   value 0 until data is actually written into the gap."
    *
-   * We can conform to the first part, but not the second.  But return EINVAL
+   * We can conform to the first part, but not the second. But return -EINVAL
```
Comment (pkarashchenko) [inline arch/arm/src/stm32h7/stm32_bbsram.c:473]:
```suggestion
   * We can conform to the first part, but not the second.  But return -EINVAL
```

File: arch/renesas/src/rx65n/rx65n_usbhost.c [hunk 5]
```diff
@@ -7649,6 +7649,7 @@ static ssize_t rx65n_usbhost_transfer(struct usbhost_driver_s *drvr,
   ed->xfrinfo = NULL;
   }
   while (0);
+
 errout_with_sem:
 
   /* rx65n_usbhost_givesem(&priv->exclsem); */
```
Comment (pkarashchenko) [inline arch/renesas/src/rx65n/rx65n_usbhost.c:7655]:
```suggestion
 errout_with_sem:
   /* rx65n_usbhost_givesem(&priv->exclsem); */
```

File: arch/sim/src/sim/vpnkit/protocol.c [hunk 6]
```diff
@@ -141,6 +141,7 @@ int really_write(int fd, uint8_t *buffer, size_t total)
     }
 
   return 0;
+
 err:
 
   /* On error: stop listening to the socket */
```
Comment (pkarashchenko) [inline arch/sim/src/sim/vpnkit/protocol.c:147]:
```suggestion
 err:
   /* On error: stop listening to the socket */
```

File: arch/xtensa/src/esp32/esp32_spiflash.c [hunk 7]
```diff
@@ -1570,9 +1570,7 @@ static ssize_t esp32_read(struct mtd_dev_s *dev, off_t offset,
   finfo("esp32_read()=%d\n", ret);
 #endif
 
-error_with_buffer:
-
-  return (ssize_t)ret;
+  return ret;
```
Comment (pkarashchenko) [inline arch/xtensa/src/esp32/esp32_spiflash.c:1573]: Notice: Maybe ret should be changed to ssize_t?

File: drivers/bch/bchdev_driver.c [hunk 8]
```diff
@@ -261,7 +261,7 @@ static off_t bch_seek(FAR struct file *filep, off_t offset, int whence)
    *   point, subsequent reads of data in the gap shall return bytes with the
    *   value 0 until data is actually written into the gap."
    *
-   * We can conform to the first part, but not the second.  But return EINVAL
+   * We can conform to the first part, but not the second. But return -EINVAL
```
Comment (pkarashchenko) [inline drivers/bch/bchdev_driver.c:264]:
```suggestion
   * We can conform to the first part, but not the second.  But return -EINVAL
```

File: drivers/eeprom/i2c_xx24xx.c [hunk 9]
```diff
@@ -497,7 +497,7 @@ static off_t ee24xx_seek(FAR struct file *filep, off_t offset, int whence)
    *   this point, subsequent reads of data in the gap shall return bytes
    *   with the value 0 until data is actually written into the gap."
    *
-   * We can conform to the first part, but not the second.  But return EINVAL
+   * We can conform to the first part, but not the second. But return -EINVAL
```
Comment (pkarashchenko) [inline drivers/eeprom/i2c_xx24xx.c:500]:
```suggestion
   * We can conform to the first part, but not the second.  But return -EINVAL
```

File: arch/arm/src/cxd56xx/cxd56_gnss.c (hunk 1 above)
Comment (xiaoxiang781216) [reply arch/arm/src/cxd56xx/cxd56_gnss.c:2788]: Done.

File: arch/arm/src/cxd56xx/cxd56_gnss.c (hunk 2 above)
Comment (xiaoxiang781216) [reply arch/arm/src/cxd56xx/cxd56_gnss.c:3073]: Done.

File: arch/arm/src/stm32h7/stm32_bbsram.c (hunk 4 above)
Comment (xiaoxiang781216) [reply arch/arm/src/stm32h7/stm32_bbsram.c:473]: The length will exceed the limitation of 79/line.

File: arch/arm/src/stm32f7/stm32_bbsram.c (hunk 3 above)
Comment (xiaoxiang781216) [reply arch/arm/src/stm32f7/stm32_bbsram.c:427]: Yes, but it make the length > 79char/line

File: arch/renesas/src/rx65n/rx65n_usbhost.c (hunk 5 above)
Comment (xiaoxiang781216) [reply arch/renesas/src/rx65n/rx65n_usbhost.c:7655]: checkpatch.sh report this warning: 
arch/renesas/src/rx65n/rx65n_usbhost.c:7654:1: error: Missing blank line before comment found

File: arch/sim/src/sim/vpnkit/protocol.c (hunk 6 above)
Comment (xiaoxiang781216) [reply arch/sim/src/sim/vpnkit/protocol.c:147]: ditto

File: arch/xtensa/src/esp32/esp32_spiflash.c (hunk 7 above)
Comment (xiaoxiang781216) [reply arch/xtensa/src/esp32/esp32_spiflash.c:1573]: Done.

File: drivers/bch/bchdev_driver.c (hunk 8 above)
Comment (xiaoxiang781216) [reply drivers/bch/bchdev_driver.c:264]: ditto

File: drivers/eeprom/i2c_xx24xx.c (hunk 9 above)
Comment (xiaoxiang781216) [reply drivers/eeprom/i2c_xx24xx.c:500]: ditto

**Comment by xiaoxiang781216** [comment]
@pkarashchenko could you take look again?

File: arch/arm/src/stm32f7/stm32_bbsram.c (hunk 3 above)
Comment (pkarashchenko) [reply arch/arm/src/stm32f7/stm32_bbsram.c:427]: yeah, but newline is an option :)

**Review by pkarashchenko — APPROVED** [review]

**Comment by xiaoxiang781216** [comment]
@pkarashchenko ping:).

**Pull request merged**